src/main/java/com/commercetools/sync/products/ProductSyncOptionsBuilder.java

```diff
@@ -47,14 +47,14 @@ public ProductSyncOptionsBuilder syncFilter(@Nonnull final SyncFilter syncFilter
     }
 
     /**
-     * Set option that indicates whether sync process should create price channel of given key when it doesn't exists
+     * Set option that indicates whether sync process should create a price channel of given key when it doesn't exist
      * in a target project yet. If set to {@code true} sync process would try to create new price channel of given key,
-     * otherwise sync process would log error and fail to process draft with given price channel key.
+     * otherwise the sync process would log error and fail to process draft with the given price channel key.
```

Comment:
```suggestion
     * otherwise the sync process would log an error and fail to process the draft with the given price channel key.
```

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/319/commits/fb76943019802e0f1d325148d729534c88d9c75f

Reply:
@ahmetoz you forgot `"the" draft` 😉 

Tip: you can commit the suggestion automatically in the github ui.

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/319/commits/e81623d476095facd9982ef4117d4e57077f6338